### PR TITLE
New Features

### PR DESCRIPTION
**Pull request by njam3**
Hey @vweevers, I've added 2 features to share-dialog that I found useful in my site.
- Do not append the ? when no parameters are passed in (this is useful when using share-dialog for a custom page)
- Ability to name the opened window something other than `sharer` (ex. `_blank` to always force a new window)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,14 +35,14 @@ ShareDialog.prototype.get = function() {
       s = decodeURIComponent(s)
     return encodeURIComponent(s)
   })
-  return this._url + '?' + params
+  return this._url + ((params!='') ? '?' + params : '');
```
Comment (vweevers) [inline index.js]: For readability, I suggest to lose the parentheses and put some distance between the question marks:

``` js
return params ? this._url + '?' + params : this._url
```

**Comment by vweevers** [comment]
One small nitpick, other than that it looks good

**Comment by njam3** [comment]
Updated :+1:

**Comment by vweevers** [comment]
Published as 0.3.0.

**Pull request closed without merging**